### PR TITLE
Angular side fixes for DDP-6044

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
https://broadinstitute.atlassian.net/browse/DDP-6044 and https://broadinstitute.atlassian.net/browse/DDP-6046

Opening PDF on separate tab easier to do on study configuration. Separate PR in other repo for that.
https://github.com/broadinstitute/ddp-study-server/pull/742

### CODE REVIEW

File: ddp-workspace/projects/ddp-rarex/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ const routes: Routes = [
     canActivate: [IrbGuard],
   },
   {
-    path: RoutePaths.TermsAndConditions,
+    path: RoutePaths.TermsOfUse,
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-rarex/src/app/app-routing.module.ts:81]: Ticket asked to change link in footer from Terms And Conditions to Terms of Use. Decided that needed to change URL to keep things consistent on front-end.

File: ddp-workspace/projects/ddp-rarex/src/app/components/footer/footer.component.html [hunk 2]
```diff
@@ -13,6 +14,7 @@
         <a
           *ngIf="link.external"
           [href]="link.routePath"
+          [target]="link.openNewTab ? '_blank' : '_self'"
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-rarex/src/app/components/footer/footer.component.html:17]: Ticket requested terms of and privacy policy links open in separate tab. Extended nicely generic template here for that.

File: ddp-workspace/projects/ddp-rarex/src/app/components/footer/footer.component.ts [hunk 3]
```diff
@@ -20,14 +21,17 @@ export class FooterComponent implements OnInit {
...
-      routePath: RoutePaths.TermsAndConditions,
+      translationKey: 'Footer.Links.TermsOfUse',
+      routePath: RoutePaths.TermsOfUse,
+      openNewTab: true
     },
     {
       translationKey: 'Footer.Links.PrivacyPolicy',
       routePath: RoutePaths.PrivacyPolicy,
+      openNewTab: true
     },
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-rarex/src/app/components/footer/footer.component.ts:35]: Self-explanatory.

File: ddp-workspace/projects/ddp-rarex/src/app/router-resources.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ export enum RoutePaths {
   Activity = 'activity',
   Activities = 'activity/:instanceGuid',
   Error = 'error',
-  TermsAndConditions = 'terms-and-conditions',
+  TermsOfUse = 'terms-of-use',
   PrivacyPolicy = 'privacy-policy',
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-rarex/src/app/router-resources.ts:13]: The actual angular url. not specified in ticket, but once you change the label the old URL did not make sense.

File: ddp-workspace/projects/ddp-rarex/src/assets/i18n/en.json [hunk 5]
```diff
@@ -14,7 +14,7 @@
   "Footer": {
     "Links": {
       "Home": "RARE-X.org",
-      "TermsAndConditions": "Terms and Conditions",
+      "TermsOfUse": "Terms of Use",
       "PrivacyPolicy": "Privacy Policy"
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-rarex/src/assets/i18n/en.json:18]: The requested label.

File: ddp-workspace/projects/ddp-rarex/src/styles/_activity.scss [hunk 6]
```diff
@@ -363,3 +363,15 @@ p.question-sub {
...
+
+.boolean-answer-HD_DEVELOPMENT_CONCERNS_PATIENT {
+  mat-radio-group {
+    display: flex !important;
+    flex-direction: column !important;
+  }
+  mat-radio-button {
+     margin-top: 0 !important;
+     margin-bottom: 10px !important;
+   }
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-rarex/src/styles/_activity.scss:375]: The one item for DDP-6046 Ticket requested that options in this question have vertical alignment.
Found that I needed to tweak margins to make options look consistent with the other ones in same survey.
Before (from ticket):
Screen Shot 2021-05-10 at 3 46 11 PM
After:
Screen Shot 2021-05-10 at 3 47 50 PM

File: ddp-workspace/projects/ddp-rarex/src/styles/_common.scss [hunk 7]
```diff
@@ -117,3 +117,9 @@ table {
     vertical-align: top;
   }
 }
+
+ddp-sign-in-out {
+  button {
+    padding: 5px !important;
+  }
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-rarex/src/styles/_common.scss:124]: Ticket complained that Login/Logout was difficult to click.
Underneath, the signin-signout component is a button. The inherited padding was 0. I tried to fatten it out, 5px all around.
Note that one consequence is that the dark bar where the login/logout/dashboard links show has also been fattened, but still looks consistent with bar in footer.
On top is the new fatter bar with the easier to click login/logout button, on the bottom the old version:
Screen Shot 2021-05-10 at 3 56 08 PM

**Review by yufengwng — APPROVED** [review]

**Pull request merged**